docs/modules/spi.md

```diff
@@ -148,7 +148,7 @@ device:transfer(txdata)
 `trans` table containing the elements of the transaction:
 
 - `command` data for command phase, amount of bits was defined during device creation, optional
-- `address` data for address phase, amount of bits was defined during device creation, optional
+- `addr` data for address phase, amount of bits was defined during device creation, optional
```

Comment:
Is this the string literal used here?

https://github.com/nodemcu/nodemcu-firmware/blob/1b98a42002e2988897daaca81edf853e56edbf6e/components/modules/spi_master.c#L102-L103

If so then `command` above should be renamed to `cmd`.